### ISSUE TITLE
E5901:  (no-member) is not supressed in 0.10

### ISSUE THREAD

**Issue by Sheshagiri** [nelfin/pylint-protobuf#17]
Following is the code that I'm trying to lint 
```python
from google.protobuf import struct_pb2
class ProtobufUtilsTestCase(unittest.TestCase):
    def test_struct_to_dict(self):
        values = {
            'foo': 'bar',
            'int': 123,
            'float': 123.45,
            'nested': {
                'a': [1, 2, 3],
                'c': 'd',
                'f': {
                    'g': {
                        'h': 'j',
                        'm': [{
                            'a': '1',
                            'd': True
                        }]
                    }
                },
                'h': {
                    'a': 100,
                    'b': 123.456,
                }
            },
        }
        struct = struct_pb2.Struct()
        struct.update(values)  # pylint: disable=no-member
```
with version `0.10` last line throws the following error
```
pylint -E --load-plugins pylint_protobuf --rcfile=./../lint-configs/.pylintrc tests/ common/
************* Module tests.unit.test_protobuf_utils
tests/unit/test_protobuf_utils.py:41:8: E5901: Field 'update' does not appear in the declared fields of protobuf-generated class 'struct_pb2.Struct' and will raise AttributeError on access (protobuf-undefined-attribute)
Traceback (most recent call last):
  File "/codefresh/volume/waas/common/.tox/lint/bin/pylint", line 10, in <module>
    sys.exit(run_pylint())
```

However, I have tested with `0.7` and it works fine. I'm a newbie to the python world and please let me know if there is anything that I missed or if any more information is required.

Thanks a lot for `pylint-protobuf`.

**Comment by Kami** [comment]
It looks like the plugin now throws ``protobuf-undefined-attribute`` error in such scenario, so we need to update our pragma to use ``pylint: disable=protobuf-undefined-attribute`` until this scenario is supported as well.

But overall, it looks like the latest version with changes in 34b4056202eacdbd4cf7749bf23e2d710a57b96e includes support for most built it types which is great.

Having said that, it looks like there are some other issues in the latest version as well:

```bash
Traceback (most recent call last):
  File "/home/kami/.virtualenvs/waas-py3.7/bin/pylint", line 10, in <module>
    sys.exit(run_pylint())
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint/__init__.py", line 20, in run_pylint
    Run(sys.argv[1:])
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint/lint.py", line 1628, in __init__
    linter.check(args)
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint/lint.py", line 943, in check
    self._do_check(files_or_modules)
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint/lint.py", line 1075, in _do_check
    self.check_astroid_module(ast_node, walker, rawcheckers, tokencheckers)
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint/lint.py", line 1158, in check_astroid_module
    walker.walk(ast_node)
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint/utils.py", line 1303, in walk
    self.walk(child)
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint/utils.py", line 1300, in walk
    cb(astroid)
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint_protobuf/__init__.py", line 585, in visit_importfrom
    self._import_node(node, node.modname)
  File "/home/kami/.virtualenvs/waas-py3.7/lib/python3.7/site-packages/pylint_protobuf/__init__.py", line 591, in _import_node
    assert issubset(self._scope, new_scope)
AssertionError
```

I will dig in and open a separate issue for that.

**Comment by nelfin** [comment]
Hi @Sheshagiri, thanks for the report. It looks like this is actually caused by fixing another issue, e4e32b8 fixed an issue with missing warnings on modules from-imported from a package which "revealed" this issue. I'll fix my code addressing WKTs to handle this case.

@Kami, if you could figure out what code is triggering that assertion and raise an issue that'd be fantastic. (I only left it in since I thought it wouldn't get triggered, so some invariant I thought was true turns out to be false)

**Comment by nelfin** [comment]
@Kami, if you're happy depending on a pre-release I've released the changes in 18f0e77 as `0.11.dev1`. I'll prepare a `0.11` release soon.
